lib/factory_bot/declaration/association.rb

```diff
@@ -22,9 +22,20 @@ def ==(other)
       private
 
       def build
+        ensure_factory_is_not_a_declaration!
+
         factory_name = @overrides[:factory] || name
         [Attribute::Association.new(name, factory_name, [@traits, @overrides.except(:factory)].flatten)]
       end
+
+      def ensure_factory_is_not_a_declaration!
+        if Declaration === @overrides[:factory]
```

Comment:
I don't encounter `===` very often. I'm having a hard time using a search engine to find some documentation for it.

Is there an English counterpart that has the same semantics?

Is this line equivalent to `@overrides[:factory].is_a?(Declaration)`?

Reply:
https://ruby-doc.org/core-2.6.3/Module.html#method-i-3D-3D-3D

`is_a?` is definitely more expressive. I will change it.